### PR TITLE
Update trexwiki

### PR DESCRIPTION
**Pull request by TriX1**
Add new group co

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2589,7 +2591,9 @@
 			),
 		),
 		'+trexwiki' => array(
-			'Founder' => array(
+			'Co' => array(
```
Comment (ImBoPhil) [inline LocalSettings.php]: you need to close this array

File: LocalSettings.php [hunk 2]
```diff
@@ -2814,17 +2819,19 @@
...
```
Comment (ImBoPhil) [inline LocalSettings.php]: this array needs to be closed as well

**Comment by TriX1** [comment]
@ImBoPhil can you repair. Thanks because mysignal low to load destop version thanks :)

**Comment by ImBoPhil** [comment]
Redone in #876

**Comment by TriX1** [comment]
@ImBoPhil Thanks very much

**Pull request closed without merging**